src/store/index.js

```diff
@@ -0,0 +1,42 @@
+import { createStore } from "vuex";
+import createPersistedState from "vuex-persistedstate";
+import SecureLS from "secure-ls";
+
+//encrypt and decrypt the localStorage
+let localStorage = new SecureLS({ isCompression: true });
+
+export default createStore({
+  state: {
+    phoneNumber: null,
+  },
+  getters: {
+    getUserPhoneNumber: (state) => {
```

Comment:
Two points
- Getters are used just like computed variables. If you need to compute some value on top of an existing data property, you would use computed variables in a normal .vue component. Same case here. If you want to compute some value on top of a state variable, then you should use a getter. So ideally, in this case, a getter is not really needed. You can safely remove it and just use state variable instead.


- getters are like the computed variables but for store. So you can assume this to be a computed variable. And if you remember, the naming of computed variables is a little different from methods. If you were to use a getter here in this case, you can safely use the name `userPhoneNumber` instead of `getUserPhoneNumber`